es/deploy/README.md

```diff
@@ -179,26 +179,26 @@ Esto va a descargar una copia de tu código en PythonAnywhere. Compruébalo escr
         ├── settings.py
         ├── urls.py
         └── wsgi.py
-    
+
 
 ### Crear un virtualenv en PythonAnywhere
 
 Tal y como hiciste en tu propio ordenador, puedes crear un virtualenv en PythonAnywhere. En la consola Bash, escribe:
 
     20:20 ~ $ cd my-first-blog
-    
+
     20:20 ~ $ virtualenv --python=python3.4 myvenv
     Running virtualenv with interpreter /usr/bin/python3.4
     [...]
     Installing setuptools, pip...done.
-    
+
     20:20 ~ $ source myvenv/bin/activate
-    
-    (myvenv)20:20 ~ $  pip install django==1.8 whitenoise
+
+    (myvenv)20:20 ~ $  pip install django==1.10 whitenoise
```

Comment:
Again, we would need Spanish translators to make sure the 1.8 -> 1.9 differences are handled correctly.